dags/fivetran_acoustic.py

```diff
@@ -180,10 +180,10 @@ def _generate_acoustic_report(
         <EXPORT_COLUMNS>
             {columns}
         </EXPORT_COLUMNS>
-        <DATE_START>{date_start}</DATE_START>
+        <DATE_START>datetime(2024, 3, 13)</DATE_START>
```

Comment:
This would mean that each report generation would be expanding on each run and contain data starting March 13th 2024. The main issue with this is that over time this will mean that the report generated is larger and larger on each run and eventually we might start potentially experiencing issues due to the file size and Fivetran will continuously reprocess the same rows (incurring unnecessary additional cost).

If we're happy to continue processing one day worth of changes captured by acoustic then we should just update the start_date of the DAG itself.

Reply:
https://github.com/mozilla/telemetry-airflow/blob/b8a686ba0963385fb4a653d3a0bee793226562d5/dags/fivetran_acoustic.py#L210

Reply:
good catch thank you
I now change the start_date for all DAGs ... if that is not a good idea I will just move the new DAG out of this dag generation thing that is so annoying me right now

Reply:
If you prefer, you can also just create a new DAG definition file with just the suppression export.

Reply:
I prefer the least amount of effort right now :)